test/translate/common_test.cc

```diff
@@ -0,0 +1,36 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "lib/translate/common.h"
+
+#include "gtest/gtest.h"
+
+namespace {
+
+/*
+   Tests the logging methods
+*/
+
+const char* actual_buf = nullptr;
+
+void callback(const char* buf) { actual_buf = buf; }
```

Comment:
These should be defined inside the scope of the test. While a function definition like `callback` can't exist there, you can instead use a lambda expression with an empty capture list, e.g. `[](const char* buf) { ... }`. You could use a static local defined in the test scope from inside the lambda, but better yet would be to just place the `EXPECT_EQ` directly inside the lambda.